modules/ytp-assets-common/src/less/components/categories.less

```diff
@@ -60,4 +60,13 @@
       }
     }
   }
+
+  a {
+    color: #2a6ebb;
```

Comment:
Variable `@opendata-brand-primary` from `less/variables/palette.less` could be used instead of hardcoded color